src/redisearch_rs/fnv/src/lib.rs

```diff
@@ -49,7 +49,7 @@ impl Fnv32 {
     /// [offset basis]: http://www.isthe.com/chongo/tech/comp/fnv/#FNV-param
     #[inline]
     #[must_use]
-    pub fn with_offset_basis(offset_basis: u32) -> Fnv32 {
+    pub const fn with_offset_basis(offset_basis: u32) -> Fnv32 {
```

Comment:
We should either merge this PR right away or move this to another PR and merge it now as CI is currently broken.

(I still don't understand how this regression passed the CI and managed to be merged).

Reply:
it's now in https://github.com/RediSearch/RediSearch/pull/7124